AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -176,11 +183,32 @@ - (void)drawInContext:(CGContextRef)context bounds:(CGRect)bounds;
   LOG(@"%@, shadowInsetBounds = %@",self, NSStringFromCGRect(shadowInsetBounds));
   
   [[self context] performBlockWithLockedTextKitComponents:^(NSLayoutManager *layoutManager, NSTextStorage *textStorage, NSTextContainer *textContainer) {
+    
+    NSTextStorage *scaledTextStorage = nil;
+    BOOL isScaled = (self.currentScaleFactor > 0 && self.currentScaleFactor < 1.0);
+
+    if (isScaled) {
```

Comment:
@rcancro my main objection to this is accessing the property twice :).  What specifically were you worried about, though? I think it is a fine practice to create the local variable isScaled, but you might instead create "scaleFactor" (for property access) and put the checks in the if () directly, with a brief comment about what the scenario is designed to detect.